assessments/projects/rook/self-assessment.md

```diff
@@ -0,0 +1,374 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+For a detailed guide with step-by-step discussion and examples, check out the free 
+Express Learning course provided by Linux Foundation Training & Certification: 
+[Security Assessments for Open Source Projects](https://training.linuxfoundation.org/express-learning/security-self-assessments-for-open-source-projects-lfel1005/).
+
+# Self-assessment outline
```

Comment:
All web content should only have a single H1 entry (#)

Reply:
removed second h1 entry here: [ee390db](https://github.com/dhauss/tag-security/commit/ee390db799dfd7245bf65fa995675fe746123939)